lib/tasks/projects.rake

```diff
@@ -40,14 +42,16 @@ namespace :projects do
 
     enqueued_projects_sum = 0
     project_ids_to_check.in_batches(of: batch_size) do |project_ids_batch|
+      enqueued_projects_sum += project_ids_batch.count
```

Comment:
I noticed the batch count was reset to 0 after being iterated through, so the logs were inaccurate. I have moved it here so the count is correct.